### PR TITLE
Move logging config to separate yaml file, add file handlers for logs…

### PR DESCRIPTION
**Pull request by naleba97**
…, add Docker volume for logs to Dockerfile

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 
 bot = commands.Bot(command_prefix='z!')
 
+LOGS_DIRECTORY = 'logs'
+
 extensions = [
     'cogs.misc',
-    'cogs.tagging',
-    'cogs.twitcasting'
+    'cogs.tagging'
```
Comment (gweng88) [inline bot.py:15]: twitcasting deleted

File: bot.py [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 import os
 
 bot = commands.Bot(command_prefix='z!')
 
+LOGS_DIRECTORY = 'logs'
+
 extensions = [
     'cogs.misc',
-    'cogs.tagging',
-    'cogs.twitcasting'
```
Comment (gweng88) [inline bot.py:12]: accidental delete?

File: logging.conf.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    level: ERROR
+    formatter: default
+    filename: logs/errors.log
+    maxBytes: 10485760
+    backupCount: 20
+    encoding: utf8
+
+loggers:
+  zhenpai:
+    level: DEBUG
```
Comment (gweng88) [inline logging.conf.yaml:34]: what does this do?

Comment (gweng88) [reply logging.conf.yaml:34]: nvm

File: bot.py (hunk 2 above)
Comment (naleba97) [reply bot.py:12]: yeah, forgot to add it back

**Pull request merged**